### DISCUSSION TITLE
set surround not working for me as expected

### DISCUSSION THREAD

**Discussion by classicalguss** [JetBrains/ideavim#293] [Q&A]
I have put set surround in ideavimrc, but when I try it using cs, I get very unpredictable behaviour. For example, the following code

```
ghassan


<img id="logo" src="{{ env("APP_URL")."/images/shababMallLogo.png" }}"/>
```

If I placed my cursor in ghassan and did `csw"` I get the following

`"ghassan"<img id="logo" src="{{ env("APP_URL")."/images/shababMallLogo.png" }}"/>`

Why did it eat away my new lines and stuck <img with the word ghassan? Am I missing something?

Also another example

This
`<a id="languageChanger" class="shababMallFont" href="?lang={{ App::getLocale() == "en" ? "ar" : "en" }}">`

If I placed my cursor in the middle of "ar" and did this command `cs"'` I get the following
`<a id="languageChanger" class="shababMallFont" href="?lang={{ App::getLocale() == "en" ? "'ar'en" }}">`

Can someone please help?

**Comment by AlexPl292** [comment]
Hi! This seems to be a bug in IdeaVim. I've filed an issue in YouTrack: https://youtrack.jetbrains.com/issue/VIM-2286

**Comment by bunglegrind** [comment]
any news on this issue? cheers
